docs/api/blockchain/introduction.mdx

```diff
@@ -53,7 +53,8 @@ next page:
 http get $(API_URL)/v1/blocks?cursor=eyJiZWZvcmUiOjQzOTIwMX0
 ```
 
-Which will return the next page of results and a next `cursor` for the following page.
+Which will return the next page of results and a next `cursor` for the following page. A cursor will
```

Comment:
Don't know if you knew about hemmingwayapp but its pretty awesome. Great for when writing documentation as it gives you a score on what grade level is needed to understand this. Of course always aiming for a lower score. I tend to aim for grade 8 and below. 

This got a grade 5 so even a 5th grader could understand 😀
 `https://hemingwayapp.com/`